src/TextArea.tsx

```diff
@@ -0,0 +1,102 @@
+import styled, {css} from 'styled-components'
+import React, {TextareaHTMLAttributes, ReactElement} from 'react'
+import {TextInputBaseWrapper} from './_TextInputWrapper'
+import {FormValidationStatus} from './utils/types/FormValidationStatus'
+import sx, {SxProp} from './sx'
+
+export type TextAreaProps = {
+  /**
+   * Apply inactive visual appearance to the TextArea
+   */
+  disabled?: boolean
+  /**
+   * Indicates whether the TextArea is a required form field
+   */
+  required?: boolean
+  /**
+   * Indicates whether the TextArea validation state
+   */
+  validationStatus?: FormValidationStatus
+  /**
+   * Block
+   */
+  block?: boolean
+  /**
+   * Allows resizing of the textarea
+   */
+  resize?: 'none' | 'both' | 'horizontal' | 'vertical'
+} & TextareaHTMLAttributes<HTMLTextAreaElement> &
+  SxProp
+
+const StyledTextArea = styled.textarea<TextAreaProps>`
+  border: 0;
+  font-size: inherit;
+  font-family: inherit;
+  background-color: transparent;
+  -webkit-appearance: none;
+  color: inherit;
+  width: 100%;
+  resize: both;
+
+  &:focus {
+    outline: 0;
+  }
+
+  ${props =>
+    props.resize &&
+    css`
+      resize: ${props.resize};
+    `}
+
+  ${props =>
+    props.disabled &&
+    css`
+      resize: none;
+      cursor: not-allowed;
+    `}
+
+  ${sx};
+`
+
+/**
+ * An accessible, native textarea component that supports validation states.
+ * This component accepts all native HTML <textarea> attributes as props.
+ */
+const TextArea = React.forwardRef<HTMLTextAreaElement, TextAreaProps>(
+  (
+    {
+      value,
+      disabled,
+      sx: sxProp,
+      required,
+      validationStatus,
+      rows = 7,
```

Comment:
```suggestion
      rows = 4,
```

7 seems a bit big for the default

Reply:
It was the closest to the previous `124px` in the Figma file, 4 is basically going back to default. Is that okay?

Reply:
Yea, that's ok